spec/features/budget_transaction_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'BudgetTransactions', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'hamid', email: 'hdm@gmail.com', password: '123456', password_confirmation: '123456',
+                        id: 1)
+    @budget_transaction = BudgetTransaction.create(name: 'test', amount: 23, author_id: @user.id)
+    @group = Group.create(author_id: @user.id, name: 'test', icon: 'http://test.com')
+    @group_transaction = GroupTransaction.create(budget_transaction: @budget_transaction, group: @group)
+    sign_in @user
+    visit group_budget_transaction_path(@group, @budget_transaction)
+  end
+
+  context 'Get /Show' do
+    it 'header should contain the text "Transaction"' do
+      within('header') do
+        expect(page).to have_content('Transaction')
+      end
+    end
+
+    # rubocop:disable Layout/LineLength
```

Comment:
- [ ] I highly recommend not to disable your linters since we should solve the linters errors to have a clean, dry code base so kindly remove this and try to solve the issues not to disable them ✨🔥🚀

Reply:
Thank you for pointing it out. I have removed the disable linter lines and fixed the linter error